src/workspace/workspace-wrapper.component.css

```diff
@@ -9,15 +9,37 @@
   display: block;
   animation: fade_in_show 0.3s;
   width: 50vw;
+  height: 80%;
+  overflow-y: scroll;
 }
 
 @keyframes fade_in_show {
   0% {
     opacity: 0;
     transform: scale(0);
   }
+
   100% {
     opacity: 1;
     transform: scale(1);
   }
 }
+
+#optionsBar {
+  width: 100% !important;
+  background-color: #eff1f2;
+  position: fixed;
+  z-index: 2;
+}
+
+@media only screen and (min-width: 769px) {
+  #cardBody {
+    margin-top: 5vh;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  #cardBody {
```

Comment:
in CSS, the `#` selector is used for `id` attribute while the `.` selector is used for `class` attribute. For consistency, use the class attribute instead as we have across the app.